Dockerfile

```diff
@@ -11,36 +12,40 @@ LABEL maintainer="chbmb"
 
 RUN \
   echo "**** install runtime packages ****" && \
+  # update to PHP 8.2 from archived linuxserver/docker-cops version
   apk add --no-cache --upgrade \
-    php81-ctype \
-    php81-dom \
-    php81-gd \
-    php81-intl \
-    php81-opcache \
-    php81-phar \
-    php81-pdo_sqlite \
-    php81-zip && \
+    # libxml2 \
+    php82-dom \
+    php82-gd \
+    php82-intl \
+    # php82-json \
+    php82-mbstring \
```

Comment:
```suggestion
```
This is already included in the base https://github.com/linuxserver/docker-baseimage-alpine-nginx/blob/master/Dockerfile